src/OmniSharp.Roslyn.CSharp/Services/RoslynFeaturesHostServicesProvider.cs

```diff
@@ -12,7 +14,7 @@ public class RoslynFeaturesHostServicesProvider : IHostServicesProvider
         public ImmutableArray<Assembly> Assemblies { get; }
 
         [ImportingConstructor]
-        public RoslynFeaturesHostServicesProvider(IAssemblyLoader loader)
+        public RoslynFeaturesHostServicesProvider(IAssemblyLoader loader, OmniSharpOptions options, IOmniSharpEnvironment env)
```

Comment:
To separate concerns, this should probably be added to another `IHostServicesProvider` rather than tacked onto the `RoslynFeaturesHostServicesProvider`, since this is specifically about loading the Roslyn features assemblies. Note that `OmniSharpWorkspace` aggregates multiple `IHostServicesProviders`.